src/modules/rlm_sql/drivers/rlm_sql_cassandra/rlm_sql_cassandra.c

```diff
@@ -818,7 +817,7 @@ do {\
 			       cass_cluster_set_max_concurrent_creation(inst->cluster, inst->spawn_max));
 	}
 
-	if (delay) cass_cluster_set_reconnect_wait_time(inst->cluster, fr_time_delta_to_msec(inst->spawn_retry_delay));
+	cass_cluster_set_reconnect_wait_time(inst->cluster, fr_time_delta_to_msec(inst->spawn_retry_delay));
```

Comment:
use FR_CONF_OFFSET_IS_SET, and check the bool value

Reply:
@arr2036 Check the old code https://github.com/FreeRADIUS/freeradius-server/commit/04026a0df54c9752540fe2cc5e67ac1e57522f29#diff-627b3e727957fd2ac985c189ed299e11L822 so, create an extra variable? because I configured the `dflt` value as `2.0`, therefore it always will be _true_.

Reply:
Users can set `spawn_retry_delay=""`.

A different way to fix this is to set

```
{ FR_CONF_OFFSET("spawn_retry_delay", FR_TYPE_TIME_DELTA | FR_TYPE_REQUIRED, rlm_sql_cassandra_t, spawn_retry_delay), .dflt = "2.0" },
```
which means it's always set.  And, that if users set `spawn_retry_delay=""`, then it's disallowed.

Reply:
@alandekok got it. ready updated.